README.md

```diff
@@ -23,6 +23,41 @@ scoop bucket add platformsh https://github.com/platformsh/homebrew-tap.git
 scoop install platform
 ```
 
+### Bash installer
+
+Use the bash installer for an automated installation, using the most preferable way for your system.
+
+```console
+curl -fSsl https://raw.githubusercontent.com/platformsh/cli/main/installer.sh | bash
+```
+
+The installer is configurable using the following environment variables:
+
+* `PLATFORMSH_CLI_INSTALL_LOG` - the install log file
```

Comment:
as they are intended to be used inline / very temporarily, perhaps these don't need the prefix

it would be nice to remove references to platform/platform.sh where possible to keep the white-label-ability - same as https://github.com/platformsh/legacy-cli/blob/main/dist/installer.php

Reply:
(I don't mean I need you to remove "platform.sh" everywhere in the script yet, but losing the variable prefix seems like it would help in that direction)

Reply:
Good idea, I don't have any hard feelings from dropping the prefix, done!